examples/basic-crud-app/README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Check out the live app
 
-Not available yet. You can open [in dev mode with StackBlitz](https://stackblitz.com/fork/github/mui/mui-toolpad/tree/master/examples/basic-crud-app) in the meantime.
+https://basic-crud-app-production.up.railway.app/prod/
```

Comment:
Shall we add a link for this?

```suggestion
[Open example](https://basic-crud-app-production.up.railway.app/prod/)
```

Reply:
I used the same pattern as https://mui.com/toolpad/examples/npm-stats/, but I agree we should not show direct links. Changing it.